bin/helios-master

```diff
@@ -34,7 +34,14 @@ if [ $JMXPORT ]; then
           -Dcom.sun.management.jmxremote.authenticate=false )
 fi
 
-exec java \
+# Allow the path to the java installation to be specified in the JAVA environment variable.
+# This can be useful if you want to run helios-agent using a different version of Java than the
+# default version on the host (possibly set up through update-alternatives)
+if [ -z $JAVA ]; then
+    JAVA=java
+fi
```

Comment:
I know I'm a bit late to the game here but for future reference there's BASH fanciness for this:
```bash
messremb(negz@~)$ echo $JAVA
messremb(negz@~)$ echo ${JAVA:-/usr/bin/java}
/usr/bin/java
messremb(negz@~)$ export JAVA=/usr/bin/megajava
messremb(negz@~)$ echo ${JAVA:-/usr/bin/java}
/usr/bin/megajava
```